test/scripts/openshift-ci/setup-llm.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+PROJECT_ROOT="$(find_project_root "$SCRIPT_DIR")"
+KSERVE_DEPLOY="${1:-true}"
+
+# Get OpenShift server version and execute appropriate script
```

Comment:
Follow-up: do we benefit from the comments explain what the code below does? I would remove them as they do not match the actual flow (and even if some of them are now they can quickly get inaccurate)